### PR TITLE
DDF-2137 - separated the spatial connected sources into their own fea…

### PR DESCRIPTION
**Pull request by kcwire**
…ture which is uninstalled by default. Also made a simple CSW configuration which should be used for DDF based federation.
#### What does this PR do?
separated the spatial connected sources into their own feaure which is uninstalled by default. Also made a simple CSW configuration which should be used for DDF based federation.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@lcrosenbu @peterhuffer @millerw8 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@jaymcnallie
@kcwire
#### How should this be tested?
Verify itests.
#### Any background context you want to provide?
Connected Sources are rarely used (if at all) and cause confusion. This change doesn't remove the functionality but moved it to a new feature to try and prevent some of that confusion.
#### What are the relevant tickets?
DDF-2137
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: catalog/spatial/csw/spatial-csw-source/src/main/resources/OSGI-INF/metatype/metatype.xml [hunk 1]
```diff
@@ -14,16 +14,19 @@
 -->
 <metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
 
-    <OCD name="CSW Federated Source" id="Csw_Federated_Source" description="CSW Federated Source">
+    <OCD name="CSW Specification Profile Federated Source" id="Csw_Federated_Source"
```
Comment (lcrosenbu) [inline catalog/spatial/csw/spatial-csw-source/src/main/resources/OSGI-INF/metatype/metatype.xml]: What does the "Specification Profile" portion of this new naming convention mean, or how does this indicate that it no longer includes the Connection Source stuff?

Comment (kcwire) [inline catalog/spatial/csw/spatial-csw-source/src/main/resources/OSGI-INF/metatype/metatype.xml]: This was mostly me trying to provide more information as to when one should be used over the other.  The `Specification Profile` is meant to be the generic implementation that should interoperate with any CSW service that adheres to the CSW 2.0.2 specification.  I also added a simple configuration for the DDF CSW profile.

**Comment by jaymcnallie** [comment]
Built against a clean repo. Itests passed and verified new config options in the config admin.  Verified Connected Source feature is disabled by default. 👍

**Comment by peterhuffer** [comment]
LGTM

**Pull request merged**